custom_components/adjustable_bed/beds/okin_uuid.py

```
@@ -550,11 +558,20 @@ async def stop_all(self) -> None:
+            await self.write_command(
+                self._build_command(self._remote.flat),
+                repeat_count=100,
+                repeat_delay_ms=300,
+            )
+        finally:
+            try:
+                await self.write_command(
+                    self._build_command(0),
+                    cancel_event=asyncio.Event(),
+                )
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)